client/src/main.jsx

```diff
@@ -2,9 +2,31 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App.jsx';
 import './index.css';
+import { createContext,useState } from 'react';
+
+const config = {
+  initialColorMode: 'light',
+  useSystemColorMode: false,
+};
+export const Context = createContext();
```

Comment:
Please export the context to it's individual file and then import it into the main file to maintain readability